packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.ts

```diff
@@ -1634,7 +1609,7 @@ class HoverCardBase extends BaseComponent<IHoverCardProps, IHoverCardState> {
     cardOpenDelay: number;
     expandedCardOpenDelay: number;
     instantOpenOnClick: boolean;
-    openHotKey: KeyCodes;
+    openHotKey: number;

```

Comment:
Oh these are default props! I think the number literal it was previously is fine, tbh. I didn't realize these were default props because of Github folding..